draft-ietf-quic-transport.md

```diff
@@ -2571,6 +2584,10 @@ the data limits set by its peer.  The cryptographic handshake stream, Stream 0,
 is exempt from the connection-level data limits established by MAX_DATA.  Stream
 0 is still subject to stream-level data limits and MAX_STREAM_DATA.
 
+If a STREAM frame is received and the UNI bit does not match the UNI bit of
+previous STREAM frames for that stream id, the connection MUST be closed with
+QUIC_INVALID_FRAME_DATA.
```

Comment:
This addresses my previous comment.

nit: stream ID 

However,  I wonder if a stronger statement could be made. I interpret this new text to effectively say that "A stream MUST be unidirectional or bidirectional for its complete lifetime. An endpoint MUST send all STREAM frames for a stream ID with the UNI bit set to the same vale. An endpoint that receives a STREAM frame with a non-matching UNI bit MUST close the connection with QUIC_INVALID_FRAME_DATA".